docs/graphql/manual/getting-started/index.rst

```diff
@@ -52,7 +52,7 @@ Get started using an existing database
    :titlesonly:
    :hidden:
 
```

Comment:
(For my own education) What are hidden TOC's for?

Reply:
This is what will show in the navigation (sidebar). Since we only want to have it in the sidebar and not also listed at this point in the page, we say it's hidden.